package/content/example.php

```diff
@@ -0,0 +1,140 @@
+﻿<?php
```

Comment:
Could we reduce or remove the provided demo files?
Could we also share them between the Docker images and the packages?

@ginifizz will take a look to the graphic design part.

Reply:
I've reduced the demo files to one index.php and the two images. I've changed the frankenphp image to use [this](https://frankenphp.dev/img/logo_darkbg.svg) svg but with the white swapped to the purple from the other images.